omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -503,7 +515,17 @@
         <ComponentRef Id="MSVCRUNTIME" />
       <?endif ?>
     </Feature>
-
+    <?if $(var.IncludeSysprobe) = true ?>
+      <!-- don't install by default (indicated by level 0) -->
+      <Feature Id="WindowsNP" 
+               Level="100"  
```

Comment:
I find the comment a bit confusing given that level is set to 100 afterwards, what does it mean?

Reply:
fixed.